auto_round/export/export_to_awq/utils.py

```diff
@@ -403,6 +365,7 @@ def _get(model, self_modules):
                 return _get(model, tmp_self_modules)
             except:
                 continue
-        assert model_type in MODEL_LAYERS_BY_MODEL_TYPE, f"Model type {model_type} not supported"
+        warnings.warn(f"Model type {model_type} not supported")
```

Comment:
Model type {model_type} may be not supported by auto_awq, please checkout the auto_awq repository.